src/org/zaproxy/zap/extension/selenium/resources/help/contents/intro.html

```diff
@@ -50,22 +51,24 @@
 		<tr>
 			<td>Internet Explorer</td>
 			<td>ie</td>
-			<td>Requires IEDriverServer, if not on the system's PATH, it can be set in the
-				options. For more information on IEDriverServer refer to the <a
+			<td>The following version is known to work: 11 (older versions might work too).
+				Requires IEDriverServer, if not on the system's PATH, it can be set in the options.
+				For more information on IEDriverServer refer to the <a
 				href="https://code.google.com/p/selenium/wiki/InternetExplorerDriver">IEDriverServer
 					website</a> (see footer note for caveat when using Internet Explorer).
 			</td>
 		</tr>
 		<tr>
 			<td>Opera</td>
 			<td>opera</td>
-			<td>&nbsp;</td>
+			<td>Temporarily not working.</td>
 		</tr>
 		<tr>
 			<td>PhantomJS</td>
 			<td>phantomjs</td>
-			<td>requires PhantomJS binary, if not on the system's PATH, it can be set in the
-				options. For more information on PhantomJS and how to obtain it refer to the <a
+			<td>The following version is known to work: 2.1.1 (older version might work too).
```

Comment:
The occurrence of "version" at "...older version might...." should also be plural like the others.